aip/cloud/2604.md

```diff
@@ -0,0 +1,114 @@
+---
+aip:
+  id: 2604
+  scope: cloud
+  state: approved
+  created: 2019-06-13
+  updated: 2019-06-13
+permalink: /cloud/2604
+redirect_from:
+  - /2604
+---
+
+# Flags for arguments that take either a number or a percentage
```

Comment:
I liked your suggestion of "Numeric arguments"

Reply:
```suggestion
Numeric arguments
```